adapters/algorix/algorix.go

```diff
@@ -109,9 +109,6 @@ func preProcess(request *openrtb2.BidRequest, ext *openrtb_ext.ExtImpAlgorix) {
 				request.Imp[i].Banner = &banner
 			}
 		}
-		if len(ext.PlacementId) > 0 {
-			request.Imp[i].TagID = ext.PlacementId
-		}
 	}
```

Comment:
Hello! Just for my understanding, could you explain why you decided to reverse the change that replaces the `imp.TagID` with the `PlacementID`? I noticed that this was one of your goals in this issues description! 

> Add Algorix PlacementId Support
    add placementId and appId
    replace imp.tagid to placementId